apps/mobile/src/libs/components/link/styles.ts

```diff
@@ -0,0 +1,12 @@
+import { StyleSheet } from "react-native";
+
+import { GradientColor } from "~/libs/enums/enums";
+
+const COLOR_INDEX = 1;
+const styles = StyleSheet.create({
+	link: {
+		color: GradientColor.BLUE[COLOR_INDEX],
```

Comment:
Can we move that color to BaseColor in order to avoid getting by index? It's not so clear
@revenets what do you think?